test/unit/manipulation.js

```diff
@@ -3099,3 +3099,55 @@ testIframe(
 		} );
 	}
 );
+
+// Custom event handler test case
+
+QUnit.test( "should handle custom '_se-custom-destroy' event correctly", function( assert ) {
+
+	// Define the expected number of assertions
+
+	assert.expect( 4 );
+
+	// Set up the HTML structure by appending to the existing fixture
+
+	var fixture = jQuery( "#qunit-fixture" );
+
+	fixture.append( `
+	<div id="container">
+	  <div class="guarded removeself" data-elt="one">
+	    Guarded 1
+	  </div>
+	  <div class="guarded" data-elt="two">
+	    Guarded 2
+	  </div>
+	  <div class="guarded" data-elt="three">
+	    Guarded 3
+	  </div>
+	</div>
+	` );
+
+	// Define the custom event handler
+	jQuery.event.special[ "_se-custom-destroy" ] = {
```

Comment:
We don't need to use the same event name as in the bug report. I don't think it's a good idea as tests don't carry the same context. Let's use something generic, like `removeondestroy`.

Reply:
Done